stories/2023-02-01-Overwatch-League/OWL_story.ipynb

```diff
@@ -0,0 +1,3730 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.    owl21_reduced
</pre><p>For consistency, maybe use <code>owl21_reduced.head()</code> here (or contrariwise)?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/TuringDataStories/pull/218/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>Did the contrariwise thing</p>